filesystem/iso9660.ksy

```diff
@@ -87,6 +87,37 @@ types:
         type: u1
       - id: offset
         type: s1
+  path_table_records:
+    doc-ref: ecma-119 9.4
+    seq:
+      - id: record
+        type: path_table_record
+        repeat: eos
+  path_table_record:
+    doc-ref: ecma-119 9.4
+    seq:
+      - id: len_dir_name
+        doc-ref: ecma-119 9.4.1
+        type: u1
+      - id: len_ext_rec
+        doc-ref: ecma-119 9.4.2
+        type: u1
+      - id: loc_ext
+        doc-ref: ecma-119 9.4.3
+        type: u4le
+      - id: parent_dir_num
+        doc-ref: ecma-119 9.4.4
+        type: u2le
+      - id: dir_name
+        doc-ref: ecma-119 9.4.5
+        type: str
+        size: len_dir_name
+      - id: padding_field
+        doc-ref: ecma-119 9.4.6
+        doc: |
+          Padding field is added when len_dir_name contains an odd number
+        size: 0x1
```

Comment:
```size: len_dir_name & 1``` ?

Reply:
"fixed", going with "len_dir_name % 2 == 1"

Reply:
???

I have looked on your "fix" and don't understand why you need a separate field

Reply:
I just realise that you combined the size and if statement, it was not about the if statement itself.
The spec actually states, that is should be 0x0. ( contents: [ 0x0 ] ) ?
But that would not combine the attributes

Reply:
Do you mean that it should look like
```yaml
- id: padding_field
                 doc-ref: ecma-119 9.4.6
                 doc: |
                   Padding field is added when len_dir_name contains an odd number
                content: [0x00]
                if: len_dir_name % 2
```
?

Reply:
That would be my best interpretation of the spec.

9.4.6 Padding Field [BP (9 + LEN_DI)]
This field shall be present in the Path Table Record only if the number in the Length of Directory Identifier field is an odd number.
If present, this field shall be set to (00).